### PR TITLE
Fix unexpected leading slash with filename in package.json

### PR DESCRIPTION
**Pull request by nolsherry**
PR for checking unexpected leading slash with filename in `filename` field,
and I have added this check item in `fixFormat.js`.
@pvnr0082t , @kennynaoh , please help me review this PR, thanks!

cc @PeterDaveHello , cc #9355

### CODE REVIEW

File: test/valid-packages-test.js [hunk 1]
```diff
@@ -320,6 +320,16 @@ packages.map(function(pkg) {
...
+  packageVows[pname + ": There is a format problem in the filename field"] = function(pkg) {
+    var json = parse(pkg, true);
+    var filenamePrefixes = new RegExp("^/");
+    if (json.filename) {
+          assert.ok(!filenamePrefixes.test(json.filename),
+                  pkgName(pkg) + ": filename should not begin with /");
+    }
+  };
+
+  packageVows
```
Comment (pvnr0082t) [inline test/valid-packages-test.js]: What is this for....?

Comment (nolsherry) [inline test/valid-packages-test.js]: Sorry, It is a mistake, I will remove `packageVows`.

File: tools/fixFormat.js [hunk 2]
```diff
@@ -90,6 +90,15 @@ async.each(packages, function(item, callback) {
...
   }
+  if (pkg.filename !== undefined) {
+    var filename = pkg.filename;
+    var filenamePrefixes = new RegExp("^/");
+    if (filenamePrefixes.test(pkg.filename)) {
+      filename = (filename).replace("/", "");
+      pkg.filename = filename;
+    }
+  }
+
```
Comment (kennynaoh) [inline tools/fixFormat.js]: Please remove blank line

**Comment by nolsherry** [comment]
@pvnr0082t, I have modified the title of issue and fixed filename with leading slash of some libs in cdnjs.
@kennynaoh, I have removed the blank line, please help me review this PR, thanks！

**Comment by pvnr0082t** [comment]
@nolsherry Any update here?

**Comment by nolsherry** [comment]
Closed.

**Pull request closed without merging**